### PR TITLE
OSSMDOC-556: Distributed tracing zstream release notes 2.3.1.

### PR DESCRIPTION
**Pull request by JStickler**
Preview ->  https://deploy-preview-44459--osdocs.netlify.app/openshift-enterprise/latest/distr_tracing/distributed-tracing-release-notes.html

### CODE REVIEW

**Review by rubenvp8510 — APPROVED** [review]

File: modules/distr-tracing-rn-new-features.adoc [hunk 1]
```diff
@@ -13,10 +13,27 @@ Result – If changed, describe the current user experience.
...
+== New features and enhancements {DTProductName} 2.3.1
+
+This release of {DTProductName} addresses Common Vulnerabilities and Exposures (CVEs) and bug fixes.
+
+=== Component versions supported in {DTProductName} version 2.3.1
+
+|===
+|Component |Version
+|Jaeger
+|1.30.2
```
Comment (rubenvp8510) [inline modules/distr-tracing-rn-new-features.adoc:25]: Is this the jaeger operator version right? not the jaeger component

Comment (pavolloffay) [reply modules/distr-tracing-rn-new-features.adoc:25]: Yes the versions here are just the bundle versions. The upstream is still 1.30.0 and 0.44.0 for OTEL.

Comment (JStickler) [reply modules/distr-tracing-rn-new-features.adoc:25]: Versions confirmed in the GUI
OperatorVersions

**Review by jldohmann — APPROVED** [review]
LGTM!

**Pull request merged**